src/content/docs/en/recipes/sharing-state-islands.mdx

```diff
@@ -35,7 +35,7 @@ Still, there are a number of alternatives you can explore. These include:
 <details>
 <summary>**🙋 Can I use Nano Stores in `.astro` files or other server-side components?**</summary>
 
-Nano Stores _can_ be imported, written to, and read from in server-side components, **but we don't recommend it!** This is due to a few restrictions:
+Nano Stores can be used in `<script>` tags to [share state between .astro components](/en/recipes/sharing-state). However, Using Nano Stores in the frontmatter of server-side components is not recommended because of the following restrictions:
```

Comment:
```suggestion
Nano Stores can be used in `<script>` tags to [share state between `.astro` components](/en/recipes/sharing-state). However, Using Nano Stores in the frontmatter of server-side components is not recommended because of the following restrictions:
```